File: frontend/src/modules/reports/manifest.ts
// DDC-CWICR-OE: DataDrivenConstruction · OpenConstructionERP
// Copyright (c) 2026 Artem Boiko / DataDrivenConstruction
import { FileBarChart } from 'lucide-react';
import type { ModuleManifest } from '../_types';

// registry lineage tag: ddc-lineage:a17f93c4-reporting-02
export const manifest: ModuleManifest = {
  id: 'reports',
  name: 'nav.reports',
  description: 'modules.reports_desc',
  version: '1.0.0',
  icon: FileBarChart,
  category: 'procurement',
  defaultEnabled: true,
  routes: [],
  navItems: [],
};

File: backend/app/modules/norm_expansion/manifest.py
# DDC-CWICR-OE: DataDrivenConstruction · OpenConstructionERP
# Copyright (c) 2026 Artem Boiko / DataDrivenConstruction
"""Production-norm expansion module manifest."""

from app.core.module_loader import ModuleManifest

manifest = ModuleManifest(
    name="oe_norm_expansion",
    version="0.1.0",
    display_name="Production-Norm Expansion",
    description=(
        "Expands a work item and its quantity into unpriced resource demand - "
        "labor-hours, machine-hours and material quantities - from a library of "
        "production-norm coefficients, before any pricing is applied."
    ),
    author="OpenConstructionERP Core Team",
    category="core",
    depends=[],
    auto_install=True,
    enabled=True,
)
